### ISSUE TITLE
Method for opening port but not read?

### ISSUE THREAD

**Issue by carl689** [johnlauer/serial-port-json-server#46]
Is there a way to open the port but not read the port until prompted?

The reason I ask is I have a device that sends constant data when port is open that I would like to be able to send command to, and occasionally read back it's status.  I COULD just receive the data and ignore it, but I would obviously prefer to not send all that traffic unnecessarily over the wire.

**Comment by chilipeppr** [comment]
No, there's no ability to do that. I suppose the only thing would be to
open the port and then disconnect the websocket. When you want to read then
reconnect the websocket.
